time-elements.js

```diff
@@ -587,6 +589,33 @@
     }
   }
 
+  // Private: Format a time according to the `timezone` attribute values.
+  //
+  // el - The local-time element to format.
+  //
+  // Returns a timezone String or null if no timezone formats are provided.
+  function formatTimezone(el) {
+    // retrieve format settings from attributes
+    var options = {
+      timezone: el.getAttribute('timezone')
+    };
+
+    // remove unset format attributes
+    for (var opt in options) {
+      if (!options[opt]) {
+        delete options[opt];
+      }
+    }
+
+    // no timezone format attributes provided
+    if (Object.keys(options).length === 0) {
+      return;
+    }
+
+    var match = el._date.toString().match(/\(([^)]+)\)/);
+    return match ? match[1] : '';
+  }
+
```

Comment:
Note that Chrome in a Windows environment can intermittently throw exceptions on `new Intl.DateTimeFormat`. If you wish to use it, always wrap it in a `try..catch` block:

```js
try {
  var format = new Intl.DateTimeFormat()
} catch (e) {
  if (!(e instanceof RangeError)) {
    throw e
  }
}
```

If there was an exception, the timezone information is unavailable, so you should simply not render any timezone string.